pkg/resources/statefulsets/log.go

```diff
@@ -158,9 +158,11 @@ func NewForLogDb(t *miniov2.Tenant, serviceName string) *appsv1.StatefulSet {
 			Containers:         []corev1.Container{logDbContainer(t)},
 			RestartPolicy:      corev1.RestartPolicyAlways,
 			SchedulerName:      t.Scheduler.Name,
-			SecurityContext:    t.Spec.Log.Db.SecurityContext,
 		},
 	}
+	if t.Spec.Log.Db != nil {
+		dbPod.Spec.SecurityContext = t.Spec.Log.Db.SecurityContext
+	}
```

Comment:
this is already being evaluated on line 167, you can just add the operation there

Reply:
you are right, just updatd the PR